Directory.Build.props

```diff
@@ -13,7 +13,7 @@
 
     <IsBenchmarkProject Condition="$(MSBuildProjectName.EndsWith('.Performance')) OR $(RepoRelativeProjectDir.Contains('perf')) OR $(RepoRelativeProjectDir.Contains('benchmarkapps'))">true</IsBenchmarkProject>
     <IsSpecificationTestProject Condition="$(MSBuildProjectName.EndsWith('.Specification.Tests'))">true</IsSpecificationTestProject>
-    <IsUnitTestProject>false</IsUnitTestProject>
+    <IsUnitTestProject Condition=" '$(IsUnitTestProject)' == '' ">false</IsUnitTestProject>
```

Comment:
See comments in PR description about ensuring Helix.props is imported. Arcade SDK sets `$(IsTestProject)` based on `$(IsUnitTestProject)` and a few other properties. A couple of `*.npmproj` test projects don't follow our naming conventions and previously set `$(IsTestProject)` _after_ this file was imported.